lib/mayaUsd/utils/util.cpp

```diff
@@ -1918,6 +1918,8 @@ VtValue _ParseArgumentValue(const std::string& value, const VtValue& guideValue)
     // The export UI only has boolean and string parameters.
     if (guideValue.IsHolding<bool>()) {
         return VtValue(TfUnstringify<bool>(value));
+    } else if (guideValue.IsHolding<float>()) {
```

Comment:
Missing float handling was making it impossible to set the frameStride value from Python.

Reply:
Does this hide the double in the line 1923

Reply:
No, the type must natch exactly